docs/limitations.md

```diff
@@ -26,6 +26,7 @@ Fortunately, as in theory records don't carry any relations information, usually
 
 > *If you don't emulate relations in your data schemas, you're safe.*
```

Comment:
I think Kinto isn't supporting constrained relations on purpose, mainly to keep things simple. This decision could be revisited though, we'll probably talk about this at the upcoming London ww.